standard/classes.md

```diff
@@ -3089,9 +3087,10 @@ The use of *accessor_modifier*s is governed by the following restrictions:
 - For a property or indexer that has no `override` modifier, an *accessor_modifier* is permitted only if the property or indexer has both a get and set accessor, and then is permitted only on one of those accessors.
 - For a property or indexer that includes an `override` modifier, an accessor shall match the *accessor_modifier*, if any, of the accessor being overridden.
 - The *accessor_modifier* shall declare an accessibility that is strictly more restrictive than the declared accessibility of the property or indexer itself. To be precise:
-  - If the property or indexer has a declared accessibility of `public`, the *accessor_modifier* may be either `protected internal`, `internal`, `protected`, or `private`.
-  - If the property or indexer has a declared accessibility of `protected internal`, the *accessor_modifier* may be either `internal`, `protected`, or `private`.
-  - If the property or indexer has a declared accessibility of `internal` or `protected`, the *accessor_modifier* shall be `private`.
+  - If the property or indexer has a declared accessibility of `public`, the *accessor_modifier* may be either `private protected`, `protected internal`, `internal`, `protected`, or `private`.
```

Comment:
The grammar has quite a few inconsistencies like this and yes it could be made consistent (one way or the other) if you wish – some have previously been tided up.

Note: both the old and new text don't allow for the double keywords in either order, same elsewhere, is this covered somewhere that mentioning one includes the other?

Reply:
Having talked about this, it's probably simplest to keep the inconsistency - but it needs to be added at ~3073. (Both `private protected` and `protected private`.)

Reply:
And add `protected private` in line 3091 and 3092.

Reply:
For 3091-3094, I've changed it to use the accessibility declared by the modifier, rather than the modifier.